manifests/init.pp

```diff
@@ -112,6 +112,7 @@
   $service_flags                  = undef,
   $service_restart                = '/etc/init.d/nginx reload',
   $service_name                   = undef,
+  $manage_service                 = true,
```

Comment:
Please rename the parameter `service_manage` to follow the convention for the other service-related parameters.